### DISCUSSION TITLE
Decision Tree & Shiny

### DISCUSSION THREAD

**Discussion by carolinaheu** [HTAhackathon2024/.github#12] [Ideas]
### What is the problem we're trying to solve

I see that there is a package created by Nathan, hedtree, so maybe this idea is not so relevant, but I was thinking of enhancing it with a shiny app.
Also, not sure how relevant it is, but if there could be a bridge to other models, i.e. the decision tree feeding into a markov model - not sure how easy this is to make, just brainstorming here....

**Comment by n8thangreen** [comment]
Thanks @carolinaheu!
The linking DT and MM is something I've been going on about for a while now (and failed to get any grant money to do BTW 😢 )
I have bits of code that may be useful for kicking this off...

**Comment by n8thangreen** [comment]
`hedtree` was just me tidying up stuff done by Pete Dodd. I also tried to so something similar using `data.tree` objects with [this](https://github.com/n8thangreen/treeSimR).

**Comment by RobertASmith** [comment]
I've done similar things on consulting projects over the course of months - this would take a really long time so it's a good idea for a long-term project and maybe the first 2 days would help kick it off. 

It's worth noting that the DARTH group built something similar: https://github.com/DARTH-git/OpenTree

**Comment by bzamoratalaya** [comment]
Thanks all, There is also the package used by Pete Dodd in an infectious disease paper: 
[https://github.com/petedodd/HEdtree]
Oh, I see @n8thangreen used this model for his package! Thanks for sharing

I would like to add times apart from costs and probabilities

**Comment by hillalex** [comment]
`rdecision` is another nice package for decision tree modelling; https://github.com/cran/rdecision
